crates/assists/src/handlers/extract_struct_from_enum_variant.rs

```diff
@@ -117,7 +115,7 @@ fn insert_import(
         let new_syntax = insert_use(
             &scope,
             make::path_from_text(&mod_path.to_string()),
-            Some(MergeBehaviour::Full),
+            ctx.config.insert_use.merge,
```

Comment:
Since we probably *always* want to use the configured merge behavior when calling `insert_use`, it might be useful to create a `ctx.insert_use` method that passes this automatically. Can be done later though.